src/CustomerBundle/Form/Type/AddressType.php

```diff
@@ -40,6 +40,11 @@ class AddressType extends AbstractType
     protected $name;
 
     /**
+     * @var BasketInterface
+     */
+    protected $basket;
+
+    /**
```

Comment:
@sonata-project/contributors technically, this is a BC-break, but I feel we should consider it patch nonetheless.